cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/workbenchcontrol/handler/GenerateJob.java

```diff
@@ -109,4 +111,17 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
     MDC.remove(InfrastructureConstants.CORRELATION_ID);
   }
 
+  /**
+   * @param monitor
+   */
+  private void checkMonolithicTemplatesException(IProgressMonitor monitor) {
+
+    try {
+      GeneratorWrapperFactory.createGenerator(this.selection, monitor);
+    } catch (Throwable e) {
+      ExceptionHandler.handle(e, HandlerUtil.getActiveShell(this.event));
+    }
```

Comment:
Why is every exception giving you a hint that it will be monolithic configuration?

Reply:
The Problem here is that the compiler did not allow me to catch the specific Exception (DeprecatedMonolithicConfigurationException), because the method CreateGenerator throws other exceptions as well. DO you have maybe another idea ?

Reply:
It is not, but the point was here to catch the monolithic configuration exception. The other Exceptions should be handled as well.